src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1343,7 +1338,14 @@ static void WindowTileInspectorScrollmousedown(rct_window* w, int32_t scrollInde
 
     // Because the list items are displayed in reverse order, subtract the calculated index from the amount of elements
     const int16_t index = windowTileInspectorElementCount - (screenCoords.y - 1) / SCROLLABLE_ROW_HEIGHT - 1;
-    WindowTileInspectorSelectElementFromList(w, index);
+    if (screenCoords.x >= 2 && screenCoords.x <= 11)
+    { // Checkbox was clicked
+        WindowTileInspectorToggleInvisibility(index);
+    }
```

Comment:
This looks a bit... scary. Is there no “cleaner” way?

Reply:
You mean the hardcoded numbers? They're pretty much the left and right side of the checkbox, and the checkbox has 2 pixels of padding to the left. I suppose storing them in named constants would make it more clear.

Reply:
I mostly meant the fact we’re now checking screen coordinates instead of actual widgets, like before. But I guess there is no other way. Adding constants would be an improvement, though.

Reply:
These `screenCoords` are local to the list itself, starting at {0,0} in the top-left corner. All interaction with window lists use this to determine where was clicked.

Reply:
What I meant was that we used to check if `WIDX_BUTTON_INVISIBLE` was clicked. Now we’re checking coordinates.

Anyway, just add a few constants and it should be good.

Reply:
You can't have widgets inside a scroll view. This is very similar to the edit theme window, you can't use the colour widget, so it has to check coordinates and show a colour dropdown if clicked.